lib/multibuild/external.ts

```diff
@@ -56,7 +56,7 @@ export function pullExternal(
 	const opts = task.dockerOpts || {};
 	let authConfigPromise: Promise<RegistrySecret | {}> | {} = {};
 	if (opts.registryconfig) {
-		authConfigPromise = getAuthConfigObj(imageName, opts.registryconfig);
+		authConfigPromise = Promise.resolve(getAuthConfigObj(imageName, opts.registryconfig));
```

Comment:
Now that it’s always a promise you can simplify the variable type, two lines above.

Reply:
Actually it didn't make sense to wrap the result of `getAuthConfigObj` in a promise as we are already using `Bluebird.resolve` below.